Unix/share/omischema/CIM-2.32.0/Device/CIM_SCSIMultipathConfigurationCapabilities.mof

```diff
@@ -151,34 +151,34 @@ class CIM_SCSIMultipathConfigurationCapabilities : CIM_Capabilities {
 
       [Description ( 
           "Specifies whether the implementation supports "
-          "auto-failback (to re-enable paths that revert to a good "
+          "auto-fallback (to re-enable paths that revert to a good "
           "state) at the plugin level, the multipath logical unit "
-          "level, both levels or whether auto-failback is "
+          "level, both levels or whether auto-fallback is "
           "unsupported." ), 
        ValueMap { "0", "2", "3", "4", "5" }, 
-       Values { "Unknown", "No Autofailback support", 
-          "Autofailback support service-wide", 
-          "Autofailback support per logical unit", 
-          "Autofailback support per service or logical unit" }, 
+       Values { "Unknown", "No Autofallback support", 

```

Comment:
To harmonize w/ above, I'd expect `Auto-fallback` (or if not fixing the `i`, `Auto-failback`)...